commands/smurf.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+// import jssha
+const jsSHA = require("jssha")
+
+const {
+  DEFAULT_SCHEMAS,
+  validate
+} = require('./../validation.js')
```

Comment:
This doesn't matter too much, but you can just use `'../validation.js'` here. I know some other places in the codebase use `'./../validation.js'`, but I'd like to avoid it going forward.

Reply:
Updated to use `../validation.js`